### DISCUSSION TITLE
Any samples for using Native APIs on Windows/macOS in a xplat project?

### DISCUSSION THREAD

**Discussion by LITTOMA** [AvaloniaUI/Avalonia.Samples#72] [Q&A]
I want to use Bluetooth functions on both platforms, but I have no idea how. Any sample would be appreciated.

**Comment by timunie** [comment]
Planned sample but not ready yet. I don't have a due date yet but probably we can work together on it. The basic idea is to add an interface `IBluetoothService` in the main project and having either a static instance which is setup at start of the App or using DI like Microsoft DI. In that case the interface will have the needed functions and help us to access each platform we want.
